commit-msg

```diff
@@ -0,0 +1,25 @@
+#!/usr/bin/env bash
```

Comment:
Let's remove this one, then we should be good to move forward.

Reply:
removed